mainjsdoshop

```diff
@@ -2,18 +2,43 @@
 
 
 var DonutShop = function(locationName, avgMin, avgMax, avgPerCustomer) {
-  this.locationName = locationName
+  this.locationName = locationName;
   this.avgMin = avgMin;
   this.avgMax = avgMax;
-  this.avgPerCustoemr = avgPerCustomer;
-  this.myArray = [];
-  this.total = 0;
-
+  this.avgPerCustomer = avgPerCustomer;
 
 DonutShop.prototype.nutPerHour = function() {
-this.total = this.myArray
-this.myArray = Math.floor(Math.random() * (this.avgMax-this.avgMin + 1) + (this.avgMin * this.avgPerCustomer));
-}
+  this.totalSales = 0;
+  this.hourlyNutArray = [];
+  for(var i = 0; i < 12; i++) {
+  this.hourlyNutArray.push(Math.floor((Math.random() * (this.avgMax-this.avgMin + 1) + this.avgMin) * this.avgPerCustomer));
+  this.totalSales = this.totalSales + this.hourlyNutArray[i];
+  }
+
+return this.hourlyNutArray;
+  };
+
+DonutShop.prototype.render = function () {
+  var getTable = document.getElementById('donuttable');
+  var newRow = document.createElement('tr');
+
+  newRow.id = this.locationName;
+  newRow.innerHTML = this.locationName;
```

Comment:
What happens if we were to enter two stores with the same name? They would have the same id attrib, correct? How will your code react? What's the impact? 

Not a major problem in this case, but worth keeping in mind when you're dynamically creating id's or other unique attribs.